truss/constants.py

```diff
@@ -140,6 +140,10 @@
     "torchaudio",
     "async-batcher>=0.2.0",
     "pydantic>=2.7.1",
+    "tensorrt==10.0.1",
```

Comment:
are all of these reqs necessary? in my experience only one of `tensorrt` or `tensorrt-cu12` is needed